src/main/java/roboy/ros/RoslessCalls.java

```diff
@@ -0,0 +1,15 @@
+package roboy.ros;
+
```

Comment:
there is a folder called memory, so your refactored classes should live there (https://github.com/Roboy/roboy_dialog/tree/master/src/main/java/roboy/memory)

name RoslessCalls does not describe what it's actually doing and that it's related to memory
possible option is Neo4jMemoryInterface or something similar, we do not need to highlight the absence of ROS in the name

Reply:
Neo4jMemoryInterface exists already. Not sure if having Neo4jMemoryOperations call MemoryOperations is the best idea, but it describes the system better.

30c75b9